app/views/conference/_schedule_item.html.haml

```diff
@@ -5,6 +5,10 @@
     %div{ class: "elipsis break-words event-title", |
           style: "-webkit-line-clamp: #{ event_lines(@rooms) }; height: #{ event_height(@rooms) }px;"} |
       = event.title
+
+      - if event.state == 'canceled'
+        %span.label.label-danger CANCELLED
```

Comment:
Let's stick with the American spelling `canceled` for uniformity reasons with our event state name.